.github/workflows/update-version.yml

```diff
@@ -0,0 +1,39 @@
+name: Update VM version
+
+on:
+  schedule:
+    - cron:  '0 10 * * *'
```

Comment:
Why 10am? I guess GitHub is using UTC+0 so 2 hours behind us. Early merger before the daily? ;-)

Reply:
Copy paste from the original workflow.
Github is using `UTC` - https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule

Victoria Metrics is based in California US - so we could use UTC 01:00 to run it so we catch the end of a workday (18:00 PST) with possible updates.